.gitignore

```diff
@@ -39,3 +40,13 @@ frontend/!.yarn/plugins
 frontend/!.yarn/releases
 frontend/!.yarn/sdks
 frontend/!.yarn/versions
+
+# 🚫 Lock files (choose only one depending on your package manager)
```

Comment:
It's best if everyone uses Yarn for this, @ruthvik-mt. Makes development much faster if everyone is using the same package manager. Can you revert the changes to this file?